apps/datahub/src/styles.css

```diff
@@ -7,6 +7,9 @@ body {
   margin: 0;
 }
 
+.container-xs {
```

Comment:
On this comment, we should do something or let the code like this?

Reply:
You should remove the code because we try as much as possible to avoid having global CSS (which is very hard to maintain in the long run). For styling you can either use tailwind classes or, if styling is too elaborate, use the component stylesheet, which is encapsulated.

Reply:
i can remove the code but when we will reduce the screen the the title can overflow above the widget. Is it ok?

Reply:
No, we still need to have a working layout